biosimulators_utils/model_lang/smoldyn/validation.py

```diff
@@ -17,6 +19,15 @@
 )
 import os  # noqa: E402
 import tempfile  # noqa: E402
+from typing import Tuple
+
+
+class ModelValidation:
+    def __init__(self, validation: Tuple):
```

Comment:
define the fields with python types ... two of them are Optional

Reply:
Good catch. I meant to go back and fill out the type annotation as such:

`Tuple[List[str], List[str], Tuple[smoldyn.Simulation, List[str]]]`

Done!